apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -441,7 +442,8 @@ public override Task OpenAsync(CancellationToken cancellationToken)
                     {
                         localClient = await ClientPool.Default.AcquireClientAsync(
                                 _connectionStringBuilder.Credential,
-                                _connectionStringBuilder.EndPoint)
+                                _connectionStringBuilder.EndPoint,
+                                _connectionStringBuilder)
```

Comment:
This *looks* distinctly weird - but I understand it now... see my earlier comment about ideally only passing genuinely "additional" options (rather than all options including data source etc).